### DISCUSSION TITLE
Billable flag on Weekly Working Hours

### DISCUSSION THREAD

**Discussion by Mindstrial** [kimai/kimai#3712] [Questions & Answers] [answered]
According to the documentation, "Billable" flag is set by default to automatic when creating a new timesheet entry. While this is true when using "My times" screen, I've noticed that when using "Weekly working hours" Billable flag is by default set to "Yes" - is this intented behaviour? I can't find any setting to change this and honestly this feels like a bug to me.

**Comment by kevinpapst** [comment, marked as answer]
> and honestly this feels like a bug to me.

And honestly: you are right. 

If you want to help Kimai, I have a tiny wishlist for 2023.

I wish that users would:
- verify at the demo site first https://www.kimai.org/demo/
- use the search to check if a similar problem was already posted https://github.com/kimai/kimai/issues/3677
- and upgrade before creating duplicates: https://github.com/kimai/kimai/releases/tag/1.30.0

**Reply by Mindstrial** [reply]
Sorry, I did search but didn't find it - turns out I've searched only for open issues, my bad. Thank you for quick answer though, I'll be more thorough next time.
